test/gen-matter-1.test.js

```diff
@@ -186,6 +186,17 @@ test(
     expect(simpleTest).toContain(
       'ExternalAddon : This is example of test external addon helper.'
     )
+
+    // Testing base types based on xml defining them
+    expect(simpleTest).toContain('Baseline type for bitmap8 : int8u')
```

Comment:
I strongly suggest not using the word "baseline" anywhere here...  It will just confuse people.  The relevant term is "Base type" or "Base data type"; see Matter spec section "7.20.2. Derived Data Types".